qutebrowser/browser/qutescheme.py

```diff
@@ -435,6 +435,20 @@ def qute_settings(url):
     return 'text/html', html
 
 
+@add_handler('bindings')
+def qute_bindings(_url):
+    """Handler for qute://bindings. View keybindings."""
+    bindings = {}
+    defaults = config.val.bindings.default
+    modes = set(defaults.keys()).union(config.val.bindings.commands)
```

Comment:
I haven't thought about this before, but having a set here actually makes them show up in a random order. I think it'd be good to have them sorted, and force `normal` to be the first. Something like this would probably work:

```python
modes.remove('normal')
modes = ['normal'] + sorted(list(modes))
```